### ISSUE TITLE
How to import into Solutions

### ISSUE THREAD

**Issue by Mattw112** [prasath2k4/PCF-To-Do-List#1]
Hi most of the PCF gallery items I have used I come out to github, download solution, and then import the .zip into dynamics.

I only see a couple files for your solution?  How do I convert those into something I can import into Dynamics?

**Comment by jmeon** [comment]
Hi, did you find a way to import to Dynamics?

**Comment by RugerSR762** [comment]
Same.  Looks like a useful control but how do you install?

**Comment by Mattw112** [comment]
Yes finally found some blogs on the topic and used Visual Studio Developer Command prompt to run through the steps to compile to a zip file.

**Comment by rubenrga1985** [comment]
Hi, someone have the solution ??? or how can I create it???

**Comment by rubenrga1985** [comment]
> Yes finally found some blogs on the topic and used Visual Studio Developer Command prompt to run through the steps to compile to a zip file.

I finally compiled, but how works on Dynamics? How to configure it?

**Comment by moliveirapinto** [comment]
@prasath2k4 can you help here?

**Comment by jsdanie1** [comment]
Was this ever provided to anybody? This seems like a useful plugin. I've been hoping Microsoft one day would just build a PCF component out of the box that was clean like this, but here we are. lol This is exactly how I picture Task working in subgrids and not the convoluted way Dynamics sort of has it.
